### ISSUE TITLE
Demo is broken

### ISSUE THREAD

**Issue by jscissr** [PolymerElements/iron-dropdown#9]
The demo on https://elements.polymer-project.org/elements/iron-dropdown?view=demo:demo/index.html is broken; the dropdown can't be closed anymore.

**Comment by oswaldomilet** [comment]
+1

**Comment by juampi92** [comment]
+1

**Comment by jjunior187** [comment]
I had the same problem on my own app, my dropdown didn't closed after tapping outside the dropdown area. 
I found that after removing the animation-config from the dropdown it worked fine again, so maybe this problem is related to the animation-config directly...

**Comment by jjunior187** [comment]
Ok, found a quick fix, hope this is for good...

On the iron-dropdown component, if there is an animation, the opened/closed state is switched after the animation is finished by this function :


```javascript
l. 239
        _onNeonAnimationFinish: function() {
          this.$.contentWrapper.classList.remove('animating');
          if (this.opened) {
            Polymer.IronOverlayBehaviorImpl._renderOpened.apply(this);
          } else if (!this.canceled) {
            Polymer.IronOverlayBehaviorImpl._renderClosed.apply(this);
          }
        },
```

the problem is that this.canceled is true when we close the dropdown, so the fix is maybe to simply remove the `!` before `this.canceled`

**Comment by miztroh** [comment]
+1

**Comment by jscissr** [comment]
The issue was fixed in #10 but the demo is still broken. Google should update the Polymer Catalog more often.

**Comment by jscissr** [comment]
Now the Catalog is updated and everything works.
